packages/web-components/fast-element/src/directives/node-observation.ts

```diff
@@ -23,7 +23,7 @@ export interface NodeBehaviorBehaviorOptions<T = any> {
 }
 
 /**
- * Filters an array of nodes to only elements.
+ * Creates a function that can be used to filter a Node array, selecting only elements.
```

Comment:
I think this needs the `@public` dist tag

Reply:
Good call. On it...